### PR TITLE
improving build process

### PR DESCRIPTION
**Pull request by brettsam**
Similar changes to the SDK repo here: https://github.com/Azure/azure-webjobs-sdk/pull/1599

### CODE REVIEW

File: src/WebJobs.Extensions.DocumentDB/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK DocumentDB Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("1.1.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions.DocumentDB/Properties/AssemblyInfo.cs]: Should be 1.2.0 (to match the fact that we're at package version 1.2.0-beta now)? Just like in the SDK repo, I think I forgot these

File: src/WebJobs.Extensions.Http/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK Http Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions.Http/Properties/AssemblyInfo.cs]: 1.1.0

File: src/WebJobs.Extensions.MobileApps/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK Mobile Apps Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions.MobileApps/Properties/AssemblyInfo.cs]: 1.2.0

File: src/WebJobs.Extensions.SendGrid/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -1,17 +1,16 @@
...
 
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK SendGrid Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("2.1.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions.SendGrid/Properties/AssemblyInfo.cs]: 2.2.0

File: src/WebJobs.Extensions.Twilio/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -1,17 +1,12 @@
...
-
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK Twilio Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
 
-[assembly: ComVisible(false)]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions.Twilio/Properties/AssemblyInfo.cs]: 1.2.0

File: src/WebJobs.Extensions/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
 
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("2.1.0.0")]
```
Comment (mathewc) [inline src/WebJobs.Extensions/Properties/AssemblyInfo.cs]: 2.2.0

**Comment by brettsam** [comment]
Thanks. Making those changes now.

**Pull request merged**